codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameCodec.java

```diff
@@ -407,4 +417,20 @@ public void readHeaderBlockEnd() {
     public void readFrameError(String message) {
         ctx.fireExceptionCaught(INVALID_FRAME);
     }
+
+    protected void setRead(final boolean read) {
+        this.read = read;
+    }
+
+    protected ChannelHandlerContext getChannelHandlerContext() {
```

Comment:
can't the user just override `handlerAdded(...)` to get this ?

Reply:
That's true:)